pkg/kudoctl/cmd/init.go

```diff
@@ -169,7 +169,7 @@ func (initCmd *initCmd) run() error {
 			"IfNotPresent":
 			opts.ImagePullPolicy = initCmd.imagePullPolicy
 		default:
-			return fmt.Errorf("Unknown image pull policy %s, must be one of 'Always', 'IfNotPresent' or 'Never'", initCmd.imagePullPolicy)
+			return fmt.Errorf("unknown image pull policy %s, must be one of 'Always', 'IfNotPresent' or 'Never'", initCmd.imagePullPolicy)
```

Comment:
I'm not sure about the small case: this string is afaik user-facing (and is not wrapped) - shouldn't it start with a capital letter?